sympy/physics/mechanics/functions.py

```diff
@@ -552,14 +552,20 @@ def _f_list_parser(fl, ref_frame):
     """Parses the provided forcelist composed of items
     of the form (obj, force).
     Returns a tuple containing:
-        vlist: The velocity (ang_vel for Frames, vel for Points) in
+        vel_list: The velocity (ang_vel for Frames, vel for Points) in
                 the provided reference frame.
-        flist: The forces.
+        f_list: The forces.
 
     Used internally in the KanesMethod and LagrangesMethod classes.
     """
     def flist_iter():
-        for obj, force in fl:
+        for pair in fl:
+            if not pair:
+                # TODO: Handle the case where there is an empty tuple in an
```

Comment:
I think this should just error out? Since we should not support load format like `(load1, (), load2)`.